examples/change-or-remove-token-keys.js

```diff
@@ -0,0 +1,214 @@
+import {
+    AccountId,
+    Client,
+    PrivateKey,
+    Logger,
+    LogLevel,
+    PublicKey,
+    KeyList,
+    TokenUpdateTransaction,
+    TokenKeyValidation,
+    TokenCreateTransaction,
+    TokenType,
+    TokenInfoQuery,
+} from "@hashgraph/sdk";
+import dotenv from "dotenv";
+
+/**
+ * @description Change ot remove token keys
+ */
+
+async function main() {
+    // Ensure required environment variables are available
+    dotenv.config();
+    if (
+        !process.env.OPERATOR_KEY ||
+        !process.env.OPERATOR_ID ||
+        !process.env.HEDERA_NETWORK
+    ) {
+        throw new Error("Please set required keys in .env file.");
+    }
+
+    const network = process.env.HEDERA_NETWORK;
+
+    // Configure client using environment variables
+    const operatorId = AccountId.fromString(process.env.OPERATOR_ID);
+    const operatorKey = PrivateKey.fromStringED25519(process.env.OPERATOR_KEY);
+
+    const client = Client.forName(network).setOperator(operatorId, operatorKey);
+
+    // Set logger
+    const infoLogger = new Logger(LogLevel.Info);
+    client.setLogger(infoLogger);
+
+
+    const adminKey = PrivateKey.generateED25519();
+    const supplyKey = PrivateKey.generateED25519();
+    const newSupplyKey = PrivateKey.generateED25519();
+    const wipeKey = PrivateKey.generateED25519();
+    const freezeKey = PrivateKey.generateED25519();
+    const pauseKey = PrivateKey.generateED25519();
+    const feeScheduleKey = PrivateKey.generateED25519();
+    const metadataKey = PrivateKey.generateED25519();
+
+    // This HIP introduces ability to remove lower-privilege keys (Wipe, KYC, Freeze, Pause, Supply, Fee Schedule, Metadata) from a Token:
+    // - using an update with the empty KeyList;
+    const emptyKeyList = KeyList.of();
+    // - updating with an “invalid” key such as an Ed25519 0x0000000000000000000000000000000000000000000000000000000000000000 public key,
+    // since it is (presumably) impossible to find the 32-byte string whose SHA-512 hash begins with 32 bytes of zeros.
+    const unusableKey = PublicKey.fromString("0000000000000000000000000000000000000000000000000000000000000000");
```

Comment:
You can use `PublicKey.unusableKey()` here

Reply:
Thank you